cloudformation/campaign-central.json

```diff
@@ -185,6 +185,23 @@
       }
     },
 
+    "CampaignCentralKinesisPolicy" : {
+      "Type": "AWS::IAM::Policy",
+      "Properties": {
+        "PolicyName": "CampaignCentralKinesisPolicy",
+        "PolicyDocument": {
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Action": [ "kinesis:*" ],
```

Comment:
I know we always do `kinesis:*` but is this an opportunity to pass a specific kinesis stream?